go/cmd/dolt/commands/stashcmds/stash.go

```diff
@@ -236,6 +251,34 @@ func allAreUntrackedFilesInWorkingSet(ctx context.Context, roots doltdb.Roots) (
 	return true, nil
 }
 
+// workingSetContainsOnlyIgnoredTables returns true if all changes in working set are ignored  tables.
+// Note that only unstaged tables are subject to dolt_ignore (this is consistent with what git does.)
+func workingSetContainsOnlyIgnoredTables(ctx context.Context, roots doltdb.Roots) (bool, error) {
+	_, unstaged, err := diff.GetStagedUnstagedTableDeltas(ctx, roots)
+	if err != nil {
+		return false, err
+	}
+
+	ignorePatterns, err := doltdb.GetIgnoredTablePatterns(ctx, roots)
+	if err != nil {
+		return false, err
+	}
+
+	for _, tableDelta := range unstaged {
+		if !(tableDelta.IsAdd()) {
```

Comment:
Whoops, that's supposed to be the reverse: only newly created tables are considered, because those are the only tables affected by dolt_ignore. (That's how git does it.)